### ISSUE TITLE
Too Deep For DSLR

### ISSUE THREAD

**Issue by TarrynVenneTodd** [Archive-663/kodakFunsaver#1]
I am a little confused and hope Im not making a fool of myself by missing something super simple lol. I have printed the EF and EF-S mount for my Canon T5i DSLR. I find that when I go to attach the printed assembly to my camera, the body is too deep and might as well be touching my mirror in my camera. Is there something I am missing? 

Let me know if you have any solutions! Thanks.

**Comment by gmoua663** [comment]
Hey, Tarryn.

For it to work on DSLRs you need to put it in Mirror Lock up mode. The flange focal distance is too short on these disposable camera lenses to fit in front of the mirror like a standard lens. Once in Mirror Lock Up you won't be able to see through the viewfinder so you'll have to print this: https://github.com/Archive-663/kodakFunsaver/tree/main/viewFinder and cannibalize the viewfinder from the Kodak Funsaver. 

https://www.dummies.com/article/home-auto-hobbies/photography/cameras/canon/how-to-enable-mirror-lockup-on-the-rebel-t5i-162131/

It's a risk, so if you're unsure of using it I'd play it cautiously and not use it. I've used my old Canon Rebel XT to test the lens and it works fine for me.

**Comment by gmoua663** [comment]
Oof, I thought I mentioned it in the description but it looks like I omitted it from this version. I'll update the need for Mirror Lock up on DSLRs.

Apologies.

**Comment by Dankeshane01** [comment]
Hi gmoua663

Very cool project, but doesn't seem to work with many cameras. 

I am having a similar issue, but it is largely because I didn't understand what a mirror lock up was before starting this project. 

Does my T7i have software Mirror Lock Up, yes, but like you described with some other modern models, the viewfinder mirror drops again between photos. 

While this is for EF and EF-S Mounts, it likely won't work with most cameras from the last 15 years unless they specifically have hardware mirror lockup. I have tried with the T7i and T2i neither will work unfortunately.

**Comment by gmoua663** [comment]
@Dankeshane01 Yeah, it's something Canon does that I can't really help. I have other mounts that are better suited for the rehousing.
